tests/award.js

```diff
@@ -377,7 +376,6 @@ module.exports = {
                             .end();
                         done();
                     })
-                done();
```

Comment:
That's interesting. I know for sure that the test gets those values from the perform function before the test completes though because the test would not be able to proceed all the way to the end without some of the values calculated in those async functions. Without the done, the test was just hanging even after completion.